src/table/utils/handle-accessibility.js

```diff
@@ -1,7 +1,11 @@
+export const getCellElement = (rootElement, cellCoord) =>
+  rootElement.getElementsByClassName('sn-table-row')[cellCoord[0]]?.getElementsByClassName('sn-table-cell')[
+    cellCoord[1]
+  ];
+
 export const findCellWithTabStop = (rootElement) => rootElement.querySelector("td[tabindex='0'], th[tabindex='0']");
 
-export const updateFocus = ({ focusType, rowElements = [], cellCoord = [], providedCell = undefined }) => {
-  const cell = providedCell || rowElements[cellCoord[0]]?.getElementsByClassName('sn-table-cell')[cellCoord[1]];
+export const updateFocus = ({ focusType, cell }) => {
```

Comment:
how about 
`export const updateFocus = ( focusType, cell ) => {...}`

Reply:
hmm actually I think it helps with the readability, since there are multiple places where the call is nested with another function call, e.g. `updateFocus({ focusType: keyboard.active ? 'focus' : 'blur', cell: getCellElement(rootElement, focusedCellCoord) });`

Reply:
aha, i see.